build.gradle

```diff
@@ -102,7 +102,7 @@ ext {
 }
 
 group = "mesosphere"
-version = "0.7.10-SNAPSHOT"
+version = "0.7.11-SNAPSHOT"
```

Comment:
Good point. I'll bump this to 0.8.0, as this changes PlanManager interface, and introduces new ones.